static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -47,8 +47,29 @@ const setAdTestCookie = (): void => {
     }
 };
 
+const showHiringMessage = (): void => {
+    try {
+        if (config.get('switches.weAreHiring')) {
```

Comment:
Maybe we should disable in dev to reduce console clutter

Reply:
I'd prefer that 👍